src/platform/forms-system/src/js/components/ReviewCardField.jsx

```diff
@@ -167,9 +167,23 @@ export default class ReviewCardField extends React.Component {
       'vads-u-margin-x--0',
     ].join(' ');
 
-    const buttonClasses = ['vads-u-margin-top--1', 'vads-u-width--auto'].join(
-      ' ',
-    );
+    const updateButtonClasses = [
+      'update-button',
+      'usa-button-primary',
+      'vads-u-margin-top--1',
+      'vads-u-margin-right--1p5',
+      'vads-u-width--auto',
+    ].join(' ');
+
+    const cancelButtonClasses = [
+      'cancel-button',
+      // keeping secondary style, but it has a shadow box outline; removed by
+      // inline styling. And we can't use `va-button-link` because when hovered,
+      // it removes all padding & add a background color (using !important)
+      'usa-button-secondary',
+      'vads-u-text-decoration--underline',
+      'vads-u-width--auto',
+    ].join(' ');
```

Comment:
`va-button-link` makes a button appear as a link, if that's helpful

Reply:
Yes, I tried that but sadly the `va-button-link` adds a hover state that removes _all_ padding and adds a slightly different grey color background, making it look bad.
![link-button](https://user-images.githubusercontent.com/136959/80252263-98ec3c80-863c-11ea-9320-d89790e1c9e6.png)